forms-flow-api/src/formsflow_api/app.py

```diff
@@ -49,7 +52,13 @@ def create_app(run_mode=os.getenv("FLASK_ENV", "production")):
     logs = logging.StreamHandler()
 
     logs.setFormatter(CustomFormatter())
-    app.logger.handlers = [logs]
+    if not os.path.exists("logs"):
+        os.makedirs("logs")
+    log_file = "logs/forms-flow-webapi.log"
+    file_handler = CustomTimedRotatingFileHandler(
+        log_file, when="d", interval=1, backupCount=7
```

Comment:
Please add this as env vars, so that default values can be overriden.

Reply:
Done